webtransport/csp-fail.https.window.js

```diff
@@ -14,6 +14,18 @@ promise_test(async t => {
   document.head.appendChild(meta);
 
   let wt = new WebTransport(webtransport_url('custom-response.py?:status=200'));
-  await promise_rejects_dom(t, 'SecurityError', wt.ready, 'ready promise should be rejected');
-  await promise_rejects_dom(t, 'SecurityError', wt.closed, 'closed promise should be rejected');
+
+  // Sadly we cannot use promise_rejects_dom as the error constructor is
+  // WebTransportError rather than DOMException.
+  for (const name of ["ready", "closed"]) {
+    try {
+      await wt[name];
+      test.unreached_func(`${name}: should have rejected promise`);
+    } catch (e) {
+      assert_true(e instanceof WebTransportError);
+      assert_equals(e.name, 'WebTransportError', `${name}: WebTransportError`);
```

Comment:
More informative failure if we check name first: assert_equals: WebTransportError expected "WebTransportError" but got "SecurityError"
```suggestion
      assert_equals(e.name, 'WebTransportError', `${name}: WebTransportError`);
      assert_true(e instanceof WebTransportError);
```